src/routes/raids/ViewRaid.tsx

```diff
@@ -2,12 +2,68 @@ import React, { useEffect, useState } from 'react'
 import firestore from '#utils/firebase'
 import { useParams } from 'react-router-dom'
 
+import styled from '@emotion/styled'
+
 import UserStatBlock from '#components/user-stat-block'
 import Emoji from '#components/emoji'
 import LoadingSpinner from '#components/loading-spinner'
 
 type State = 'loading' | 'success' | 'not-found' | 'error'
 
+const StatsView = styled.main`
+  height: 100vh;
+  display: grid;
+  grid-template-rows: auto 1fr;
+  > * {
+    padding: 1rem;
+  }
+`
+
+const Header = styled.header`
+  z-index: 1;
```

Comment:
hmm, why z-index plz?

Reply:
Good question :D I'll remove it since it seems unnecessary.